assets/css/_dialogs.scss

```diff
@@ -215,26 +215,147 @@
   font-weight: bold;
 }
 
-.email-sign-in-dialog {
-  text-align: center;
-  width: 500px;
+#icon-twitter path {
+  fill: white;
+}
+
+#icon-facebook path:last-child {
+  fill: white;
+}
+
+#icon-facebook path:first-child {
+  fill: #3b579d;
+}
+
+.sign-in-dialog {
+  text-align: left;
+  width: 300px;
+  min-height: 300px;
```

Comment:
Remove the `min-height` so that when only one or two sign-in options are available, the awkward white space goes away:

<img width="409" alt="screenshot 2018-07-08 23 30 56" src="https://user-images.githubusercontent.com/2553268/42429474-1e9aec0e-8307-11e8-8ae3-35f44b8c0373.png">